### PR TITLE
Bump version for release 0.0.8

### PR DESCRIPTION
**Pull request by christiam**

### CODE REVIEW

File: docs/source/issues.rst [hunk 1]
```diff
@@ -55,7 +59,18 @@ ElasticBLAST uses cloud storage to temporally store query sequences and internal
...
 --------------------------------------------
 
 When doing a synchronous search (submitted with the ``--sync`` option), ElasticBLAST is continuously probing for search status to know when the search is done. When the status check times out ElasticBLAST interprets it as search failure and shuts down the cluster.
+
+.. _too_many_jobs:
+
+Too many jobs leads to a failed execution
+-----------------------------------------
+
+If the ElasticBLAST configuration leads to too many (e.g.: over 5,000)
```
Comment (boratyng) [inline docs/source/issues.rst]: This description seems vague. Do we know that there is a maximum number of jobs a kubernetes cluster can handle? If yes, then we should say that this is the maximum.

File: docs/source/issues.rst [hunk 2]
```diff
@@ -55,7 +59,18 @@ ElasticBLAST uses cloud storage to temporally store query sequences and internal
...
+kubernetes jobs created, ElasticBLAST will disconnect from the cluster and it
```
Comment (boratyng) [inline docs/source/issues.rst:75]: Is it our code that disconnects from the cluster?

File: docs/source/issues.rst (hunk 1 above)
Comment (christiam) [reply docs/source/issues.rst]: Yes, see https://cloud.google.com/kubernetes-engine/quotas

File: docs/source/issues.rst (hunk 2 above)
Comment (christiam) [reply docs/source/issues.rst:75]: I haven't investigated this, but the fact is that there is a disconnect.

Comment (boratyng) [reply docs/source/issues.rst:75]: That was my worry, if there is a disconnect we may be not deleting the cluster.

File: docs/source/issues.rst [hunk 3]
```diff
@@ -55,7 +59,18 @@ ElasticBLAST uses cloud storage to temporally store query sequences and internal
...
+.. _early_shutdown:
+
 A synchronous search may shut down too early
 --------------------------------------------
 
 When doing a synchronous search (submitted with the ``--sync`` option), ElasticBLAST is continuously probing for search status to know when the search is done. When the status check times out ElasticBLAST interprets it as search failure and shuts down the cluster.
+
+.. _too_many_jobs:
+
+Too many jobs leads to a failed execution
```
Comment (boratyng) [inline docs/source/issues.rst:71]:
```suggestion
Too many query sequences lead to a failed execution
```
Most users will not know what a job is and how to relate it to what they are doing.

File: docs/source/issues.rst (hunk 1 above)
Comment (boratyng) [inline docs/source/issues.rst]:
```suggestion
ElasticBLAST divides query sequences into batches and searches them in parallel. The `batch-len` parameter controls the size of a single batch. If the ElasticBLAST configuration leads to more than 5,000 query batches
```

Comment (boratyng) [reply docs/source/issues.rst]: Yes, this is vague too. The only applicable quota from this page seems to be 300k container per cluster. We have 3 containers per job, which gives us a lot of room. Running this:
```
kubectl get resourcequota gke-resource-quotas -o yaml
```
There is this:
```
spec:
  hard:
    count/ingresses.extensions: "100"
    count/jobs.batch: 5k
    pods: "1500"
    services: "500"
status:
  hard:
    count/ingresses.extensions: "100"
    count/jobs.batch: 5k
    pods: "1500"
    services: "500"
  used:
    count/ingresses.extensions: "0"
    count/jobs.batch: "17"
    pods: "6"
    services: "1"
```
This is probably where maximum of 5k jobs comes from.

File: docs/source/issues.rst (hunk 2 above)
Comment (christiam) [reply docs/source/issues.rst:75]: that is not the case in the case I observed, but this needs to be investigated further

File: docs/source/issues.rst (hunk 1 above)
Comment (christiam) [reply docs/source/issues.rst]: >This is probably where maximum of 5k jobs comes from.

Yes.

File: docs/source/issues.rst (hunk 3 above)
Comment (christiam) [reply docs/source/issues.rst:71]: Thanks for the suggestion but this is not accurate.

**Review by boratyng — APPROVED** [review]

**Pull request merged**